### PR TITLE
Users/tasoni/bug fix

### PR DESCRIPTION
**Pull request by tanvi-soni**

### CODE REVIEW

File: Tasks/RunDistributedTests/task.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
             "label": "Test Drop Location",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "Location where the test binaries have been dropped in the agent machine(s) as part of the 'Copy files to Machines' task. System Environment Variables can also be used in location string. e.g., '%systemdrive%\\Tests', '%temp%\\DropLocation' etc.",
+            "helpMarkDown": "Location where the test binaries have been dropped in the agent machine(s) as part of the 'Azure File Copy' task. System Environment Variables can also be used in location string. e.g., '%systemdrive%\\Tests', '%temp%\\DropLocation' etc.",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: we need both

"Copy files to Machine" or "Azure File Copy" task

File: Tasks/RunDistributedTests/task.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
...
             "groupName": "testSettings"
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: u also need to build using Gulp. check with Kavipriya.there will be files that will change after build

File: Tasks/RunDistributedTests/task.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "id": "D353D6A2-E361-4A8F-8D8C-123BEBB71028",
     "name": "RunVisualStudioTestsusingTestAgent",
     "friendlyName": "Visual Studio Test using Test Agent",
-    "description": "Run tests distributedly using multiple test agents on a lab machine group",
+    "description": "Run tests across multiple test agents on a lab machine group in a distributed fashion",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: also u need to change the task patch version (increase by one)

**Comment by bryanmacfarlane** [comment]
accept or close

**Pull request closed without merging**